src/main/java/duke/classes/Commands.java

```diff
@@ -0,0 +1,68 @@
+package duke.classes;
+
+/**
+ * Enum class for categorising the commands given to the scanner.
+ */
+public enum Commands {
```

Comment:
Would it be better to name this 'Command' instead of 'Commands'? 'Commands' may suggest to the reader that it is a collection.